cloud/azure/templates/azure_saml_ses/bin/setup.py

```diff
@@ -36,6 +36,20 @@ def pre_terraform_setup(self):
         if not self.config.use_backend_config():
             self._make_backend_override()
 
+    def get_current_user(self):
+        current_user_function = subprocess.run(
+            [
+                "/bin/bash", "-c",
+                f"source cloud/azure/bin/lib.sh && azure::get_current_user_id"
+            ],
+            capture_output=True)
+        current_user = None
+        if current_user_function:
```

Comment:
suggestion (blocking): this `if` will always be true, recommend removing and just checking `current_user` instead

Reply:
alternatively, could do `if current_user_completed_process.returncode == 0:`

Reply:
suggestion (blocking): rename `current_user_function` to `current_user_completed_process`

Reply:
ahh good call!